src/react/boosts/BoostForm.tsx

```diff
@@ -413,6 +413,15 @@ class BoostFormComponent extends React.Component<BoostFormProps, BoostFormState>
                   disabled={this.props.editMode}
                 />
                 <EndDateNotice>Your Boost will end at 11:59PM on the date selected.</EndDateNotice>
+                {!this.props.isStripeConnected && (
+                  <ConnectStripeNotice>
+                    We noticed you don't have a Stripe account set up. If you'd like to connect one, you can do that{" "}
```

Comment:
Should we mention credit cards here? Maybe something like

> *No Stripe account connected*
> Your Boost will be able to accept contributions in ETH, but if you connect a Stripe account you will be also able to accept credit card payments. If you'd like to connect Stripe you can do so from [your newsroom dashboard]. You will be prompted to create a new account if you don't have one already.

Also should that be `target="_blank"`, or does `target="[any unrecognized name]"` work?

Reply:
good catch!